plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/SettingsForm.js

```diff
@@ -190,6 +195,20 @@ Ext.define('NX.view.SettingsForm', {
   },
 
   /**
+   * Update the state of the save button
+   *
+   * @private
+   */
+  updateButtonState: function (form) {
+    var dirty = form.isDirty(),
+        valid = form.isValid();
+
+    if (form.down('button[action=save]')) {
```

Comment:
we should refrain from multiple executions of the same component query, extract a variable here please to avoid duplicate lookup and defining the actual CQ twice